### PR TITLE
Update installation instructions to include parler [fix #41]

### LINKED ISSUE #41: Readme file needs updating. Installation doesn't work.
New project. Installed Alias in the usual way... just doesn't work. Can't add Aliases. Can't add categories.
```
TemplateDoesNotExist at /en/admin/djangocms_alias/category/add/
admin/parler/change_form.html
```

```
AttributeError at /en/admin/djangocms_alias/aliascontent/add/
'NoneType' object has no attribute 'category'
```

### PR DESCRIPTION
**Pull request by marksweb**
Updates the installation instructions to include the dependency on parler, fixing #41

### CODE REVIEW

**Review by Aiky30 — CHANGES_REQUESTED** [review]
Needs a changelog entry so that we can track this coming through releases.

**Comment by marksweb** [comment]
@Aiky30 changelog entry added

File: CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ Unreleased
 ==================
 * feat: Remove Add cta and hide delete dropdown actions from AliasContent admin ChangeList
 * feat: Site field added to plugin
+* fix: Added parler to the installation instructions as an installed app
```
Comment (Aiky30) [inline CHANGELOG.rst:12]:
```suggestion
```

Comment (Aiky30) [inline CHANGELOG.rst:12]:
```suggestion
```

Needs moving to the unreleased section, I think the fork you had was not updated.

**Review by Aiky30 — CHANGES_REQUESTED** [review]
Hi Mark, 

I think your forked branch was out of date, I've updated the branch rules to force updates to make this easier in future. The changelog is now showing in the released portion. Annoyingly I can't use the GH suggestions to add this back in myself.

**Review by Aiky30 — APPROVED** [review]

**Pull request merged**